packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -141,11 +141,14 @@ class EditableText extends StatefulWidget {
   /// Creates a basic text input control.
   ///
   /// The [maxLines] property can be set to null to remove the restriction on
-  /// the number of lines. By default, it is 1, meaning this is a single-line
-  /// text field. If it is not null, it must be greater than zero.
+  /// the number of lines. By default, it is one, meaning this is a single-line
+  /// text field. [maxLines] must not be set to zero.
```

Comment:
According to your change here, it's now allowed to be negative, which I doubt is what you intended.